src/components/Book.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+const Book = ({ category, title }) => (
```

Comment:
- [ ] It accepts `Book` object as prop [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png) (You need to adapt this component to receive a book object and then restructure it inside accordingly, please do so to comply with the project requirements)

```

Reply:
The book component accepts a book object as props anymore.